.github/workflows/ci.yml

```diff
@@ -0,0 +1,37 @@
+name: Lint and unit tests
+on:
+  push:
+    branches:
+      - '**'
+    tags:
+      - 'v*'
+  pull_request:
+    branches:
+      - master
+jobs:
+  golangci:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v2
+        with:
+          version: v1.32
+          args: --enable goimports
+  tests:
+     name: Tests
+     runs-on: ubuntu-latest
+     env:
+       ACTIONS_ALLOW_UNSECURE_COMMANDS: true
```

Comment:
Is this an exception that we're making in other components, too?

Reply:
Yes adapters also have this env variable